src/config/endpoint.ts

```diff
@@ -1,2 +1,2 @@
-export const API_ENDPOINT_DEV = 'https://niftykit-dev.herokuapp.com/api'
+export const API_ENDPOINT_DEV = 'https://api-testnet.niftykit.com'
 export const API_ENDPOINT = 'https://app.niftykit.com/api'
```

Comment:
this is now `https://api.niftykit.com`

Reply:
will update it once it's ready for production, did a couple of tests with the new prod URL, using both the new format: `{NEW_PROD_URL}/drops/{collectionId}/address` and the old one `{NEW_PROD_URL}/drops/address` with the `x-api-key` and always returned a `401` status. Once this new URL is ready will also implement #7 👍